tools/quicktype-wrapper/README.md

```diff
@@ -0,0 +1,50 @@
+# quicktype-wrapper
+
+This node module that wraps the `quicktype` tool to generate code using JSON Schema.
+
+## Install
+
+This tool can be installed as a global CLI or run as a Node script. Requires Node 8+.
+
+### CLI (Recommended)
+
+Install the CLI globally:
+
+```sh
+sudo npm run cli
+```
+
+Then type `qt` to run the generator.
+
+### Node Script
+
+You can also run as a script. Build then run:
+
+```sh
+npm run compile
+npm run start
+```
+
+## Run
+
+const IN = process.env.IN || `${HOMEDIR}/Documents/github/googleapis/google-cloudevents/proto/`;
+const OUT = process.env.OUT || `${HOMEDIR}/Documents/out`;
+const L = (process.env.L || LANGUAGE.TYPESCRIPT).toUpperCase() as TARGET_LANGUAGE;
+
+
+## Develop/Watch
+
+To continually watch the script insead of building after every change,
+run this command in a separate terminal:
+
+```sh
+npm run watch
+```
+
+## Lint/Fix
+
+To fix style/lint mistakes, run:
```

Comment:
What is this to fix mistakes in - the JSON schema files?

Reply:
No, this README is in the `tools/quicktype-wrapper/` folder/project. The JSON schemas are not in this folder (in this repo, but really tangential).

Reply:
So what *would* it fix? If this is just generating code, wouldn't any style/lint issues suggest a problem with the generator or the schema?

Reply:
Fixing the schema is out of scope of this PR/folder. Any tooling to do that would not live in the `tools/quicktype-wrapper` tool.

Why do you want this feature here?

Reply:
I don't particularly - I'm trying to find out what this *does* fix mistakes in; that's what I've been asking.

Reply:
Added a more detailed comment.